domain-cc/cc-app/src/python_src/api.py

```diff
@@ -108,20 +132,27 @@ def log_contention_stats(
 
     is_multi_contention = len(claim.contentions) > 1
 
-    log_as_json(
-        {
-            "vagov_claim_id": sanitize_log(claim.claim_id),
-            "claim_type": sanitize_log(log_contention_type),
-            "classification_code": classification_code,
-            "classification_name": classification_name,
-            "contention_text": log_contention_text,
-            "diagnostic_code": sanitize_log(contention.diagnostic_code),
-            "is_in_dropdown": is_in_dropdown,
-            "is_lookup_table_match": classification_code is not None,
-            "is_multi_contention": is_multi_contention,
-            "endpoint": request.url.path,
-        }
-    )
+    logging_dict = {
+        "vagov_claim_id": sanitize_log(claim.claim_id),
+        "claim_type": sanitize_log(log_contention_type),
+        "classification_code": classification_code,
+        "classification_name": classification_name,
+        "contention_text": log_contention_text,
+        "diagnostic_code": sanitize_log(contention.diagnostic_code),
+        "is_in_dropdown": is_in_dropdown,
+        "is_lookup_table_match": classification_code is not None,
+        "is_multi_contention": is_multi_contention,
+        "endpoint": request.url.path,
+    }
+
+    if request.url.path == "/expanded-contention-classification":
+        logging_dict = log_expanded_contention_text(
+            logging_dict, contention.contention_text, log_contention_text
+        )
+
+        # log_as_json(logging_dict)
+    # else:
```

Comment:
Is this commented-out code left in deliberately?

Reply:
@gabezurita Sorry I missed this comment before merging. That was accidentally left behind, I will clean this up in a following PR.